Makefile

```diff
@@ -1,5 +1,5 @@
 build:
-	bundler exec jekyll build
+	LANG="en_US.UTF-8" LC_ALL="en_US.UTF-8" bundler exec jekyll build
```

Comment:
I think it is pretty strange that you keep having problems that require you to do this.  Is this on your machine, or on the cproof server?

Reply:
This is on the cproof machine, and started after we switched machines.